product_docs/docs/edbcloud/beta/administering_cluster/02_postgres_access.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: "Managing Postgres Access"
```

Comment:
Do we want to cover superuser access requests and limitations in this page? /cc @gbartolini